superset/views/chart/views.py

```diff
@@ -33,6 +33,7 @@ class SliceModelView(
     SliceMixin, SupersetModelView, DeleteMixin
 ):  # pylint: disable=too-many-ancestors
     route_base = "/chart"
+    class_permission_name = "Chart"
```

Comment:
This comes out okay, but the previous permission still persists.
![Screenshot from 2020-04-03 04-29-21](https://user-images.githubusercontent.com/20752142/78307515-b52c1a80-7563-11ea-9795-e552d575323c.png)
![Screenshot from 2020-04-03 04-29-49](https://user-images.githubusercontent.com/20752142/78307550-c8d78100-7563-11ea-9610-8663e784f8c4.png)